pl_examples/basic_examples/conf/models/basic.yaml

```diff
@@ -0,0 +1,9 @@
+# @package _group_
+drop_prob: 0.5
```

Comment:
the model config looks like a mix of many concepts (model, optimizer, data loader).
it might make sense to put some of those variables under different nodes.

Reply:
Started to do this breakout with optimizer, model, data...almost done